container.go

```diff
@@ -142,6 +142,8 @@ type ContainerRequest struct {
 	HostConfigModifier      func(*container.HostConfig)                // Modifier for the host config before container creation
 	EnpointSettingsModifier func(map[string]*network.EndpointSettings) // Modifier for the network settings before container creation
 	LifecycleHooks          []ContainerLifecycleHooks                  // define hooks to be executed during container lifecycle
+	LogProducer             bool                                       // enable the log producer
```

Comment:
IIUC this bool is a flag to start producing logs, right? Wdyt if we remove it and simply check that the LogConsumers slice has elements? We can assume that if the users provided consumers it's because they want to produce logs.

Reply:
Done